Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -344,9 +344,9 @@ NodePort XDP Acceleration
 Cilium has built-in support for accelerating NodePort, LoadBalancer services and
 services with externalIPs for the case where the arriving request needs to be
 pushed back out of the node when the backend is located on a remote node. This
-ability to act as a hairpin load balancer can be handled by Cilium at the XDP
```

Comment:
Could you also change the following above if you're touching this file (I wasn't fast enough to submit my review yesterday)?
```diff
- (one per eth0 and eth1 devices)
+ (one for each of devices eth0 and eth1)
```

Reply:
np, done.